### DISCUSSION TITLE
Immersun Charger

### DISCUSSION THREAD

**Discussion by dward32** [lbbrhzn/ocpp#1112] [General]
Hi, I'm using a new Immersun charger which has OCPP - J as an option and I've set it up and it's feeding to home assistant. I can start/stop the charge however I only ever get 4.8a charge and not 32a as I should be. The slider is set to 32a. Any ideas what I'm doing wrong? Thanks

**Comment by sandersr** [comment]
Most likely nothing. It might be that your charger is not supporting "chargingProfileKind":"Relative" or "chargingProfilePurpose":"ChargePointMaxProfile"

To test, I suggest the following:
go to developer tools -> services
find "Open Charge Point Protocol (OCPP): Set maximum charge rate"

feel the following yaml config:

service: ocpp.set_charge_rate
data:
  custom_profile:
    chargingProfileId: 8
    stackLevel: 0
    chargingProfileKind: Absolute
    chargingProfilePurpose: TxDefaultProfile
    chargingSchedule:
      chargingRateUnit: A
      chargingSchedulePeriod:
        - startPeriod: 0
          limit: 16

if it works, you can add it to your automations via "call service" option and raise a feature request with your charger manufacturer
